posts/2022-08-01-Increasing-glibc-kernel-requirements.md

```diff
@@ -0,0 +1,121 @@
+---
+layout: post
+title: "Increasing the glibc and kernel requirements"
```

Comment:
Maybe "Linux" should be in the title too?

Reply:
Done! I've changed kernel -> Linux kernel in the title.